### ISSUE TITLE
[a11y] 9.4.1.3 - Status messages programatically available

### ISSUE THREAD

**Issue by tbsbdr** [owncloud/web#5398]
- Context: The Web UI should be accessible according to the WCAG2.1 /BITV 2.0 guidelines. The number in the title relates to the test-steps: https://www.bitvtest.de/bitv_test/das_testverfahren_im_detail/pruefschritte.html
- "Status" relates to the evaluation scheme https://www.bitvtest.de/bitv_test/das_testverfahren_im_detail/verfahren.html#c1262 
- "a11y" is short for accessibility
- ----------------
## General notes
All pages / Popups "Apps-Menü" and "Benutzereinstellungen" (probably also other similar Popups affected): After activating the button the whole container of the displayed popup will be read out by screenreaders.
(On user settings it is the 3 links). Better solution would be to let the screenreader read out only „erweitert“ after activating the app-menue. The aria-describedby-Attribute which triggers the output should be removed.
**Note:** the issue also appears with Firefox. Edge doesn’t read out the content.

## Status: partly fulfilled

**Comment by paulcod3** [comment]
Unclear if aria-describedby needs to be removed from every OcDrop instance, needs further clarification.

**Comment by kulmann** [comment]
Went through all usages of `OcDrop`, IMO the `aria-describedby` in the way we use it can be removed entirely, because the focus order dives down into the respective `OcDrop`.

**Comment by kulmann** [comment]
Thinking about this exactly three years later together with @pascalwengerter - let's remove the the `aria-describedby` and make sure that all icon-only-buttons which toggle an OcDrop have a proper `aria-label` and an aria-hidden tooltip.
